### ISSUE TITLE
Expose Faker again

### ISSUE THREAD

**Issue by esambo** [brousalis/peteshow#8]
Faker used to be globally exposed which allowed the consuming Rails app to do things like: `Faker.Address.streetAddress()`
But the introduction of _gulp_ in #2 stopped doing this which now causes the error: `Uncaught ReferenceError: Faker is not defined` and stops Peteshow from loading.

I wouldn't mind if Faker would be scoped within Peteshow, but such a breaking change should probably be mentioned in a Changelog.md file

Affected apps based on a [GitHub search](https://git.enova.com/search?l=javascript&q=Faker.Address&ref=cmdform&type=Code) are:
- [NetCredit](https://git.enova.com/netcredit/netcredit_web_common/blob/master/vendor/assets/javascripts/peteshow.US.js)
- [Headway](https://git.enova.com/headway/headway/blob/master/vendor/assets/javascripts/peteshow.SMB.js)
- [OnStride](https://git.enova.com/onstride/onstride/blob/master/vendor/assets/javascripts/peteshow.GB.js)
- [P2P](https://git.enova.com/trogdor/p2p/blob/master/vendor/assets/javascripts/peteshow.GB.js)
- [Flex](https://git.enova.com/henchmen/flex/blob/master/vendor/assets/javascripts/peteshow.GB.js)
- [Plugin example wiki](https://github.com/brousalis/peteshow/wiki/Plugin-example)

**Comment by brousalis** [comment]
It wasn't on purpose. When Kevin did the gulp changeover, he broke that on accident.

I'll fix it shortly.

**Comment by esambo** [comment]
There seem to be some [other errors](https://enova.slack.com/archives/headway/p1420062483000067) too when trying to fill out a form with Peteshow

**Comment by brousalis** [comment]
I think its pretty safe to say kevin's gulp implementation is broken and master is dirty. I'll revert everything.

However I never released his changes to the gem itself, so the gem is still stable.

**Comment by esambo** [comment]
Yes, 0.8.6 works, and rebasing #2 onto that last commit before gulp [works fine](https://github.com/brousalis/peteshow/pull/7#issuecomment-68545051)

**Comment by itsthatguy** [comment]
Hey!

**Comment by itsthatguy** [comment]
Digging into all of this now
